src/internal/packager/sbom/viewer/viewer.js

```diff
@@ -45,55 +45,69 @@ function initData() {
             "Size",
         ],
         "data": transformedData,
-    }
+    };
 
     if (window.dt) {
-        window.dt.destroy()
+        window.dt.destroy();
     }
 
-    distroInfo.innerHTML = payload.distro.prettyName || 'No Base Image Detected'
+    distroInfo.innerHTML = payload.distro.prettyName || "No Base Image Detected";
 
     window.dt = new simpleDatatables.DataTable(artifactsTable, {
         data,
         perPage: 20,
-    })
-
+    });
 }
 
 function fileList(metadata) {
     if (metadata) {
         const list = (metadata.files || [])
             .map(file => {
-                return file.path || ''
+                return file.path || "";
```

Comment:
Not a big deal either way, but just curious if this was prettier or some linting changing these from single to double quotes? I assume this was since we introduced all the JS/TS config stuff with Zarf UI?

Reply:
This was me trying to normalize it.  I'll actually change it to what the prettierrc says to do.